static-analysis/teammates-checkstyle.xml

```diff
@@ -141,6 +141,7 @@
         <module name="SingleLineJavadoc"/>
         <module name="JavadocVariable">
             <property name="scope" value="public"/>
+            <property name="tokens" value="VARIABLE_DEF"/>
```

Comment:
Added this line to suppress missing javadoc warnings for enum values

https://github.com/checkstyle/checkstyle/issues/1669

Reply:
I think you should disable it for that file/section instead of disabling it for the whole project.

Reply:
Makes sense. I have suppressed it in `teammates-checkstyle-suppressions.xml` instead

Reply:
`https://checkstyle.sourceforge.io/config_annotation.html#SuppressWarnings` is more straight-forward? I think `teammates-checkstyle-suppressions.xml` is not supposed to be used in that way. It contains TODO works instead.

Reply:
Ahh I didn't realize. Thanks for clarifying!